Python/marshal.c

```diff
@@ -1810,8 +1809,8 @@ Functions:\n\
 \n\
 dump() -- write value to a file\n\
 load() -- read value from a file\n\
-dumps() -- write value to a string\n\
-loads() -- read value from a string");
+dumps() -- write value to a bytes object\n\
```

Comment:
Maybe “marshal value as a bytes object” or “convert value to a bytes object”. Writing _to_ a bytes object sounds like you pass the bytes object in and it is modified.